go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/carousell/Orion
 
-go 1.17
+go 1.22
```

Comment:
coverage seems reduced even when there is no change on the code. could you check

Reply:
Something seems to be off with this tool.
https://app.codecov.io/gh/carousell/Orion/pull/231/indirect-changes
In the page it mentioned 15 indirect file coverage changes, but the change% is missing.